jupyterbook/_toc.yml

```diff
@@ -0,0 +1,81 @@
+- file: content/intro
+  title: The IOOS Data Demo Center
+
+- part: Code Gallery
+  chapters:
+
+    - file: content/Code Gallery/data_access/data_access.md
+      sections:
+          - file: content/Code Gallery/data_access/2016-10-12-fetching_data.ipynb
+          - file: content/Code Gallery/data_access/2017-05-14-running_compliance_checker.ipynb
+          - file: content/Code Gallery/data_access/2016-12-20-searching_glider_deployments.ipynb
+          - file: content/Code Gallery/data_access/2017-01-18-siphon-explore-thredds.ipynb
+          - file: content/Code Gallery/data_access/2017-03-21-ERDDAP_IOOS_Sensor_Map.ipynb
+          - file: content/Code Gallery/data_access/2017-06-12-NCEI_RA_archive_history.ipynb
+          - file: content/Code Gallery/data_access/2017-07-25-HFRadar_currents.ipynb
+          - file: content/Code Gallery/data_access/2017-12-15-finding_HFRadar_currents.ipynb
+          - file: content/Code Gallery/data_access/2018-03-01-erddapy.ipynb
```

Comment:
This is tricky :-/

Currently our notebooks are listed by date and we don't need to list them anywhere to publish them. If we cannot automate the creation of this TOC we should document it. @lohithmunakala can you create a wikipage with the process that one would take to "add a new notebook?"

Reply:
I am thinking of writing a .sh file that will automate this process. But for now, sure, I will write a wikipage for manually changing the `toc.yaml` file.

Reply:
before you get too far, I started this [wiki page](https://github.com/ioos/notebooks_demos/wiki) to document some of that information. It probably needs to be expanded upon. 

@ocefpaf what was the thinking behind adding dates in the titles of the notebooks? Is this something we might want to reevaluate in this new paradigm?

Reply:
Sure will add changes to the wiki file.